### PR TITLE
Paul/saved action

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206044974374431

### CODE REVIEW

File: src/plugins/gui/providers/bityProvider.ts [hunk 1]
```diff
@@ -490,6 +499,28 @@ const completeSellOrder = async (approveQuoteRes: BityApproveQuoteResponse, core
   }
 
   const spendInfo: EdgeSpendInfo = {
+    savedAction: {
+      type: 'sell',
```
Comment (swansontec) [inline src/plugins/gui/providers/bityProvider.ts:503]: Going along with my core review, I see that you are saving a `sell` action, but not a `sellNetworkFee` action. I really don't think the `sellNetworkFee` action does anything, or makes sense to even save.

**Review by swansontec — COMMENTED** [review]
I am going to ask @Jon-edge to review this, since he has more opinions about the transaction action system.

File: src/plugins/gui/providers/paybisProvider.ts [hunk 2]
```diff
@@ -526,8 +528,32 @@ export const paybisProvider: FiatProviderFactory = {
                       console.log(`  tokenId: ${tokenId}`)
                       const nativeAmount = await coreWallet.denominationToNative(amount, displayCurrencyCode)
 
+                      const savedAction: EdgeTxActionFiat = {
+                        type: 'sell',
+                        orderId: invoice,
+                        orderUri: '',
```
Comment (Jon-edge) [inline src/plugins/gui/providers/paybisProvider.ts:534]: This prop is optional according to your core branch, should just leave it undefined for clarity

File: src/plugins/gui/providers/bityProvider.ts [hunk 3]
```diff
@@ -490,17 +499,35 @@ const completeSellOrder = async (approveQuoteRes: BityApproveQuoteResponse, core
...
+      orderId: id,
+      orderUri: '',
```
Comment (Jon-edge) [inline src/plugins/gui/providers/bityProvider.ts:505]: This prop is optional according to your core branch, should just leave it undefined for clarity

**Review by Jon-edge — APPROVED** [review]

File: src/plugins/gui/providers/paybisProvider.ts (hunk 2 above)
Comment (paullinator) [reply src/plugins/gui/providers/paybisProvider.ts:534]: Actually good you caught this. There's actually a URI I can put into this.

File: src/plugins/gui/providers/bityProvider.ts (hunk 1 above)
Comment (paullinator) [reply src/plugins/gui/providers/bityProvider.ts:503]: The `sellNetworkFee` is used for token transactions

**Review by Jon-edge — APPROVED** [review]

File: src/plugins/gui/providers/bityProvider.ts (hunk 1 above)
Comment (Jon-edge) [reply src/plugins/gui/providers/bityProvider.ts:503]: Maybe `sellNetworkFee` and `Token Sell Network Fee` is more appropriate then. I thought it was kind of strange with the "Sell Network Fee" string.

**Review by Jon-edge — CHANGES_REQUESTED** [review]

File: src/components/scenes/TransactionDetailsScene.tsx [hunk 4]
```diff
@@ -57,17 +57,15 @@ const TransactionDetailsComponent = (props: Props) => {
...
   const txActionInfo = getTxActionDisplayInfo(transaction, wallet, tokenId)
-  const txActionSplitCat = txActionInfo?.splitCategory
+  const txActionSplitCat = txActionInfo?.edgeCategory
   const txActionNotes = txActionInfo?.notes
   const txActionDir = txActionInfo?.direction
 
+  const isSentTransaction = transaction.nativeAmount.startsWith('-') || (eq(transaction.nativeAmount, '0') && transaction.isSend)
+
   // Determine direction from transaction nativeAmount if not specified in
-  // txActionInfo
```
[the remark this answers is not in the document]
Comment (paullinator) [reply src/components/scenes/TransactionDetailsScene.tsx:71]: The remaining comment is removed in a later commit anyway

**Review by Jon-edge — APPROVED** [review]

**Review by Jon-edge — APPROVED** [review]

**Review by Jon-edge — APPROVED** [review]
Last fixes make sense

**Review by Jon-edge — APPROVED** [review]

**Review by Jon-edge — APPROVED** [review]

**Pull request merged**